src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/collectionsgrid/viewmodel/CollectionsGridViewModel.kt

```diff
@@ -39,7 +39,7 @@ class CollectionsGridViewModel: ViewModel() {
     val filteredContent: ObservableList<Pair<SimpleObjectProperty<Content>, SimpleBooleanProperty>>
             = FXCollections.observableArrayList()
 
-    private var loading: Boolean by property(false)
+    private var loading: Boolean by property(true)
```

Comment:
val?

Reply:
no, it has to be a var otherwise the we get an type inference error on the next line from tornadofx